### PR TITLE
Cards wave 3

### PR DESCRIPTION
**Pull request by maxgestic**
Added Card Wave 3 by Emma

Plus some log file annoyance fix

### CODE REVIEW

**Comment by maxgestic** [comment]
Tested and working

File: resources/usa_properties/sv_properties.lua [hunk 1]
```diff
@@ -544,6 +544,18 @@ local burglarySearchItems = {
...
+    { name = "Mini", type = "tradingCard", src = { front = 'Mini.png', back = 'usarrp-trading-card-background.png' }, price = 300, weight = 1.0, quantity = 1, notStackable = true, objectModel = "p_ld_id_card_002" },
+    { name = "Nancy", type = "tradingCard", src = { front = 'Nancy.png', back = 'usarrp-trading-card-background.png' }, price = 300, weight = 1.0, quantity = 1, notStackable = true, objectModel = "p_ld_id_card_002" },
+    { name = "Richard_2", type = "tradingCard", src = { front = 'Richard_2.png', back = 'usarrp-trading-card-background.png' }, price = 300, weight = 1.0, quantity = 1, notStackable = true, objectModel = "p_ld_id_card_002" },
+    { name = "TJ", type = "tradingCard", src = { front = 'TJ.png', back = 'usarrp-trading-card-background.png' }, price = 300, weight = 1.0, quantity = 1, notStackable = true, objectModel = "p_ld_id_card_002" },
+    { name = "Tony_2", type = "tradingCard", src = { front = 'Tony_2.png', back = 'usarrp-trading-card-background.png' }, price = 300, weight = 1.0, quantity = 1, notStackable = true, objectModel = "p_ld_id_card_002" },
```
Comment (minipunch) [inline resources/usa_properties/sv_properties.lua:558]: It is true that the more we add here the less chance players have to get other types of items is

Would prob be best to handle it like we do with the lucky wheel 

Where it's only one entry but then it chooses 1 random card out of all of em to give

Comment (minipunch) [reply resources/usa_properties/sv_properties.lua:558]: Though for now this is prob fine

File: resources/usa_utils/commerce/server/sv_tebexPackageFunctions.lua [hunk 2]
```diff
@@ -86,6 +86,7 @@ TEBEX_PACKAGE_FUNCTIONS = {
             { name = "Exodia Head", type = "tradingCard", src = { front = 'exodia.png', back = 'yugioh-back.jpg' }, price = 300, weight = 1.0, quantity = 1, notStackable = true, objectModel = "p_ld_id_card_002" },
             { name = "Shooting Magestic Star Dragon", type = "tradingCard", src = { front = 'shooting-magestic-star-dragon.png', back = 'yugioh-back.jpg' }, price = 300, weight = 1.0, quantity = 1, notStackable = true, objectModel = "p_ld_id_card_002" },
             { name = "Toon Blue Eyes Dragon", type = "tradingCard", src = { front = 'toon-blue-eye-dragon.png', back = 'yugioh-back.jpg' }, price = 300, weight = 1.0, quantity = 1, notStackable = true, objectModel = "p_ld_id_card_002" },
+            { name = "Mini", type = "tradingCard", src = { front = 'Mini.png', back = 'usarrp-trading-card-background.png' }, price = 300, weight = 1.0, quantity = 1, notStackable = true, objectModel = "p_ld_id_card_002" },
```
[the remark this answers is not in the document]
Comment (maxgestic) [reply resources/usa_utils/commerce/server/sv_tebexPackageFunctions.lua:89]: Yes

File: resources/usa_properties/sv_properties.lua (hunk 1 above)
Comment (maxgestic) [reply resources/usa_properties/sv_properties.lua:558]: Yea ill make this a seperate issue and PR

**Pull request merged**